### PR TITLE
Add nielsen analytics to amp articles

### PR DESCRIPTION
**Pull request by QuarpT**
## What does this change?

Add Nielsen analytics to amp articles
See https://trello.com/c/FagOKxZ0/136-add-neilsen-analytics-to-amp-for-australia

I've put in some guards to test this against a few articles in prod

## Tested in CODE?

Yes

### CODE REVIEW

File: common/app/model/Section.scala [hunk 1]
```diff
@@ -63,3 +63,31 @@ object SectionId {
 
   def fromId(id: String): SectionId = SectionId(id)
 }
+
+object Neilsen {
+  private val sectionIdToApid = Map(
```
Comment (philmcmahon) [inline common/app/model/Section.scala]: should this be `sectionIdToAppId`? or is apid a thing in itself?

**Review by philmcmahon — APPROVED** [review]
Other than tiny thing above about what apid is, looks good

File: common/app/model/Section.scala (hunk 1 above)
Comment (QuarpT) [reply common/app/model/Section.scala]: APIDs are unique ids used by Nielsen. I'm not sure what it stands for :P

File: common/app/model/Section.scala [hunk 2]
```diff
@@ -63,3 +63,31 @@ object SectionId {
 
   def fromId(id: String): SectionId = SectionId(id)
 }
+
+object Neilsen {
```
Comment (QuarpT) [inline common/app/model/Section.scala]: There are more Guardian sections than the ones listed here. I'm still waiting for feedback from Nielsen on how the other sections should be mapped to APIDs.

**Pull request merged**